CHANGELOG.md

```diff
@@ -6,3 +6,7 @@ Inception of the Node GLSP Server.
 This project provides the Node based server component for the Eclipse Graphical Language Platform (GLSP).
 The implementation of this server is aligned with the default Java based [GLSP Server](https://github.com/eclipse-glsp/glsp-server).
 The initial initial implementation was contributed on behalf of STMicroelectronics.
+
+### Changes
+
+-   [diagram] Implement LayoutEngine API for server-side autolayouting & provide an integration package for layout engines based on ELK. [#509]() - Contributed on behalf of STMicroelectronics
```

Comment:
```suggestion
-   [diagram] Implement LayoutEngine API for server-side autolayouting & provide an integration package for layout engines based on ELK. [eclipse-glsp/glsp#509]() - Contributed on behalf of STMicroelectronics
```

Not sure if this works in markdown files. If not we should probably do normal links to the glsp repository.

Reply:
I forgot to update the changelog after I opened the PR. Typically what we do here is linking the Issue number with the corresponding PR. 
in this case [#509]((https://github.com/eclipse-glsp/glsp-server-node/pull/2)